src/api/Middleware/groups.js

```diff
@@ -0,0 +1,33 @@
+import {
+  getCustomGroupsOrdering,
+  updateCustomGroupOrdering
+} from '../Actions';
+import actionTypes from '../Actions/actionTypes';
+
+const getGuiGroupsOrdering = async (luaApi, callback) => {
+  const object = await luaApi.guiTreeOrdering();
```

Comment:
```suggestion
  const object = await luaApi.getGuiGroupsOrdering();
```

Otherwise, there is an error when loading the UI page